src/pages/[platform]/build-a-backend/troubleshooting/library-not-configured/index.mdx

```diff
@@ -0,0 +1,142 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Library Not Configured',
+  description: 'Addressing Library Not Configured error message',
+  platforms: [
+    'angular',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'vue'
+  ],
+  canonicalObjects: [
+    {
+      platforms: [
+        'angular',
+        'react',
+        'react-native',
+        'vue',
+        'nextjs',
+        'javascript'
+      ],
+      canonicalPath: '/angular/build-a-backend/troubleshooting/migrate-from-javascript-v5-to-v6/'
+    }
+  ]
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+If you are running into the `Library Not Configured` error message and have called `Amplify.configure` in your project, your Amplify API is most likely being called before `Amplify.configure`. This can happen in a few different scenarios. Below are a list of things you can check to troubleshoot this issue.
+
+## Call `Amplify.configure` in the root of your project
+
+Make sure you are calling `Amplify.configure` in the root file of your project. The root file of your app may be different depending on your frontend framework:
+
+* Vue: **src/main.ts** or **src/main.js**
+* React: **src/main.tsx** or **src/main.jsx**
+* Next:
+* Angular: **main.ts** or **main.js**
```

Comment:
Still need to update these paths and the verbiage to specify that these will be the default paths in common frameworks.